# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/snap/reftest.list
# Commit: fc2b3d6448bc
# Full Hash: fc2b3d6448bcc3575fffb1abc923e7e703aeef54
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 8c70bc92ad36 -r fc2b3d6448bc gfx/wr/wrench/reftests/snap/reftest.list
--- a/gfx/wr/wrench/reftests/snap/reftest.list	Wed Feb 23 20:49:26 2022 +0000
+++ b/gfx/wr/wrench/reftests/snap/reftest.list	Wed Feb 23 20:49:27 2022 +0000
@@ -2,3 +2,4 @@
 == transform.yaml transform.png
 platform(linux,mac) == preserve-3d.yaml preserve-3d.png
 fuzzy(128,200) == subpixel-raster-root.yaml subpixel-raster-root-ref.yaml
+platform(linux,mac) == fractional-filter.yaml fractional-filter-ref.yaml